src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -100,22 +100,27 @@ public function prepare_items() {
 		if ( 'scheduled' === $this->tab->get_slug() && ! empty( $_REQUEST['s'] ) ) {
 			// nonce check if search form submitted.
 			$nonce = isset( $_POST['s'] ) && isset( $_POST['aggregator']['nonce'] ) ? sanitize_text_field( $_POST['aggregator']['nonce'] ) :  '';
-			if ( wp_verify_nonce( $nonce,  'aggregator_' . $this->tab->get_slug() . '_request'  ) ) {
+			if ( isset( $_GET['s'] ) || wp_verify_nonce( $nonce,  'aggregator_' . $this->tab->get_slug() . '_request'  ) ) {
```

Comment:
* :no_entry_sign: Expected 1 spaces before closing parenthesis; 2 found
* :no_entry_sign: Expected 1 space after comma in argument list; 2 found

<sub>hash: 1c93f0ecc9d7de476f5b192bb702c04e</sub>

Reply:
Address what the bot menitoned.